src/frontend/components/TeacherLiveRecordingActions/index.spec.tsx

```diff
@@ -125,4 +125,29 @@ describe('<TeacherLiveRecordingActions />', () => {
       screen.queryByRole('button', { name: /stop recording/i }),
     ).not.toBeInTheDocument();
   });
+
+  it('does not render buttons if the live recording is disallowed', () => {
+    const video = videoMockFactory({
+      live_state: liveState.RUNNING,
+      allow_recording: false,
+    });
+
+    render(
+      wrapInIntlProvider(
+        <QueryClientProvider client={queryClient}>
+          <TeacherLiveRecordingActions
+            isJitsiAdministrator={false}
```

Comment:
```suggestion
            isJitsiAdministrator={true}
```
as it's not rendered if you're not a isJitsiAdminstrator your test is incomplete, If you only want to get the property allow_recording, this one should be true

Reply:
yes, you're right thanks, I changed it but not in this PR, but on the prior one.